package.json

```diff
@@ -28,47 +28,47 @@
   "author": "Felix Becker <felix.b@outlook.com>",
   "license": "ISC",
   "dependencies": {
-    "chalk": "^2.3.0",
-    "inquirer": "^5.0.0",
+    "chalk": "^2.4.1",
+    "inquirer": "^5.2.0",
     "mkdirp": "^0.5.1",
     "mz": "^2.4.0",
     "sql-template-strings": "^2.2.0",
-    "update-notifier": "^2.3.0",
+    "update-notifier": "^2.5.0",
```

Comment:
What's the point of updating these range requirements? Are we using any new features of the latest releases? Is anything not working with previous releases?

Reply:
reverted them